tests/sentry/api/endpoints/test_release_threshold_details.py

```diff
@@ -0,0 +1,73 @@
+from django.urls import reverse
+
+from sentry.models.environment import Environment
+from sentry.models.release_threshold.releasethreshold import ReleaseThreshold
+from sentry.testutils.cases import APITestCase
+
+
+class ReleaseThresholdDetailsTest(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.user = self.create_user(is_staff=True, is_superuser=True)
+
+        self.canary_environment = Environment.objects.create(
+            organization_id=self.organization.id, name="canary"
+        )
+        self.production_environment = Environment.objects.create(
+            organization_id=self.organization.id, name="production"
+        )
+        self.login_as(user=self.user)
+
+        self.basic_threshold = ReleaseThreshold.objects.create(
+            threshold_type=0,
+            trigger_type=0,
+            value=100,
+            window_in_seconds=1800,
+            project=self.project,
+            environment=self.canary_environment,
+        )
+
+    def test_invalid_threshold_id(self):
+        url = reverse(
+            "sentry-api-0-project-release-thresholds-details",
+            kwargs={
+                "organization_slug": self.organization.slug,
+                "project_slug": self.project.slug,
+                "threshold_id": 123,
+            },
+        )
+        response = self.client.get(url)
+
+        assert response.status_code == 404
```

Comment:
same nit, can switch to much shorter test request
```suggestion
        self.get_error_response(self.organization.slug, self.project.slug, 123, status_code=404)
```

Reply:
hmm... eh 🤷 

I'm indifferent.
Sure it can make the tests shorter, but it makes them more difficult to decipher and obfuscates away what the test is actually testing for